src/runtime/SrcDocIFrameController.ts

```diff
@@ -1,11 +1,11 @@
+/* eslint-disable @typescript-eslint/no-non-null-assertion */
```

Comment:
I suggest using a single-line ESLint disable directive:
`// eslint-disable-next-line @typescript-eslint/no-non-null-assertion`

instead of a block directive:
`/* eslint-disable @typescript-eslint/no-non-null-assertion */`

This scopes the rule exception to only the specific line where needed, making the code more explicit and maintainable.